DFC.ServiceTaxonomy.GraphSync/Handlers/GraphSyncContentHandler.cs

```diff
@@ -171,15 +171,15 @@ public override async Task RemovingAsync(RemoveContentContext context)
         // so we handle them at the base context level by cancelling the session
         private void Cancel(ContentContextBase context)
         {
-            _session.Cancel();
+            _session.CancelAsync().Wait();
```

Comment:
Is this safe? Sorry, im not up on the Wait method

Reply:
YesSql.ISession has dropped the synchronous Cancel() method. Rather than refactor i've added the Task.Wait() which effectively makes the CancelAsync() a synchronous call.
I do agree that it looks less than ideal and we could raise a backlog task to review and refactor if you prefer.

Reply:
Okay cool - as long as you've seen it work. In the older Framework days you always had to be very wary about deadlocks when mixing sync and async didnt you - but less of an issue with core (and the lack of the default sync context)